### PR TITLE
 Deprecate Customer entity methods

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
-_TBD_
+- [x] Deprecated `Customer` entity methods: `getFirstName`, `setFirstName`, `getLastName`, `setLastName`, `getEmail`, `setEmail`
```
Comment (slavcodev) [inline CHANGELOG.md]: Please follow `keepachangelog` principles.

**Comment by XaosSintez** [comment]
@adamaltman 
```
public function getEmail()
{
    return $this->getPrimaryAddress()->getEmails();
}
```

This will not work as client expects, because instead of `string` it will return `Email[]`.
I can refactor methods like this:
```
public function setEmail($value)
    {
        $primaryAddress = $this->getPrimaryAddress();
        $email = Email::createFromData([
            'label' => 'main',
            'value' => $value,
            'primary' => true,
        ]);

        if ($primaryAddress !== null) {
            $primaryAddress->setEmails([$email]);
        } else {
            $primaryAddress = Address::createFromData([
                'emails' => [$email],
            ]);
        }

        $this->setPrimaryAddress($primaryAddress);

        return $this;
    }
```

It will work with some additional adjustments (make `setPrimaryAddress()` to use `email` set by `setEmail()` method).

But it really looks like a hack. The best way is not to use those deprecated methods at all.

Please let me know what you think about this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ Security - in case of vulnerabilities.
 - [x] Removed deprecated `LeadSource` property: `ipAddress`
 - [x] Removed `Notification`, `EmailNotification` and `EmailNotificationTracking`.
 - [x] Deprecate setting `initialInvoiceId` property when creating a `Subscription`.
+- [x] Deprecated `Customer` entity methods: `getFirstName`, `setFirstName`, `getLastName`, `setLastName`, `getEmail`, `setEmail`
```
Comment (slavcodev) [inline CHANGELOG.md]: Why did you add this in already released version? 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +24,9 @@ _TBD_
 ### Changed
 - [x] **Upgraded minimum PHP version to 7.1**
 
+### Removed
```
Comment (slavcodev) [inline CHANGELOG.md]:
```suggestion
### Deprecated
```

**Comment by adamaltman** [comment]
If the method doesn't work, it's not deprecated, it's broken.
